src/agimus_sot/control/gripper.py

```diff
@@ -61,22 +61,20 @@ def _makeTorqueControl (self, nums, denoms):
     def _makeIntegrationOfVelocity (self):
         from dynamic_graph.sot.core.operator import Add_of_vector
         self.omega2theta = Add_of_vector (self.name + "_omega2theta")
-        self.omega2theta.setCoeff2(self.dt)
-        # self.omega2theta.sin1 <- current position
-        # self.omega2theta.sin2 <- desired velocity
-        plug (self.torque_controller.output, self.omega2theta.sin2)
+        self.omega2theta.coeffs = np.array([0, self.dt])
```

Comment:
Shouldn't it be
```suggestion
        self.omega2theta.coeffs = np.array([1, self.dt])
```

Reply:
I do not know. I did not find any initialization for the first coefficient.

Reply:
I think they were initialized with ones. The operation that this does is
![image](https://user-images.githubusercontent.com/2388209/106137808-88d53080-616b-11eb-81e6-62f23940a026.png)